lib/MiqVm/MiqRhevmVm.rb

```diff
@@ -57,6 +57,10 @@ def getCfg(_snap = nil)
       cfg_hash["#{tag}.devicetype"] = "disk"
       cfg_hash["#{tag}.filename"]   = file_path.to_s
       cfg_hash["#{tag}.format"]     = disk.format
+
+      if storage_obj.storage.type == "ISCSI" && !lv_active?(file_path.to_s)
+        `sudo lvchange -ay #{file_path.to_s}`
```

Comment:
So, then, this would be something like

```suggestion
        AwesomeSpawn.run!("sudo lvchange", :params => [:a, :y, file_path.to_s])
```

I'm not 100% sure, but I think there's a better way to run via sudo as well - can't remember if kernel.spawn has that built-in or not.

Reply:
I don't see this option in AwesomeSpawn or in Kernel.spawn's documentation